sdks/go/pkg/beam/core/graph/edge.go

```diff
@@ -235,7 +236,7 @@ func NewCoGBK(g *Graph, s *Scope, ns []*Node) (*MultiEdge, error) {
 // the shared input type.
 func NewFlatten(g *Graph, s *Scope, in []*Node) (*MultiEdge, error) {
 	if len(in) < 2 {
-		return nil, fmt.Errorf("flatten needs at least 2 input, got %v", len(in))
+		return nil, fmt.Errorf("creating new Flatten in scope %v: Flatten needs at least 2 input, got %v", s, len(in))
```

Comment:
Curiosity question,

Why do we use %v as the format string for len(in). Would not %d be more appropriate?

Reply:
From what I've seen %v is just generally used unless there's a reason not to, like applying additional formatting.

Reply:
I found this informative answer on differences (https://stackoverflow.com/questions/41159169/why-not-use-v-to-print-int-and-string). I guess it does no make much of a difference in this case because the builtin len will always return an int.